styleguide/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "ama-style-guide",
-  "version": "2.25.3",
+  "version": "2.25.4",
```

Super minor @designerneil , can you make this 2.26.0?
```suggestion
  "version": "2.26.0",
```

---

why are we jumping to 2.26.0?

---

We're in Sprint 26 right?